libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -139,6 +140,8 @@ AP_BLHeli::AP_BLHeli(void)
     AP_Param::setup_object_defaults(this, var_info);
     _singleton = this;
     last_control_port = -1;
+    // register as an ESC telemetry source
+    AP::esc_telem().add_backend(this);
```

Comment:
Looks like only three of the drivers call this function. Shouldn't all of them call it in their constructor?

Reply:
Actually this was unused so I have removed them all

Reply:
Thanks for fixing this.